editions/tw5.com/tiddlers/filters/syntax/Filter Expression.tid

```diff
@@ -34,8 +34,6 @@ If a run has:
 
 <<.tip "Compare named filter run prefix `:filter` with [[filter Operator]] which applies a subfilter to every input title, removing the titles that return an empty result from the subfilter">>
 
-<<.tip "Compare named filter run prefix `:reduce` with [[reduce Operator]] which is used to used to flatten a list of items down to a single item by repeatedly applying a subfilter.">> 
```

Comment:
Why is this deleted?

Reply:
I moved this tip into the new "Reduce Filter Run Prefix" tiddler. Would you prefer it be in both places?

Reply:
@btheado I think having it in both places is preferable.

Reply:
Thanks for the feedback. I restored the tip to both places.